MAINTAINERS.md

```diff
@@ -9,5 +6,12 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 
 | Maintainer             | GitHub ID                                         | Affiliation |
 | ---------------------- | ------------------------------------------------- | ----------- |
-| Joshua Li              | [joshuali925](https://github.com/joshuali925)     | Amazon      |
-| Rupal Mahajan          | [rupal-bq](https://github.com/rupal-bq)           | Amazon      |
```

Comment:
Are you sure for the removal of maintainers here?

Reply:
They are not listed as maintainers here: https://github.com/orgs/opensearch-project/teams/dashboards-reports/members?query=role%3Amaintainer. We may need to start an email chain to change this, but as of now this PR will get it up to date as per github